### ISSUE TITLE
Disabling unused connectors

### ISSUE THREAD

**Issue by mgarofano80** [socfortress/CoPilot#142]
It would be useful to allow the user to disable unneeded connectors as to avoid generating errors in the UI and the logs section.

**Comment by D47K-0v3771D3** [comment]
I agree.... I logged into my SOCfortress and it's been spinning hours without allowing me to change the connector because the previous Wazuh instance is deleted

**Comment by screwlooseit** [comment]
Would like to see this implemented as well.

**Comment by guilhermesferreira** [comment]
That would be amazing
